misaligned.cpp

```diff
@@ -1,21 +1,42 @@
 #include <iostream>
 #include <assert.h>
 
+namespace ColorMap {
+
+int getColorPairNumber(int majorColorIndex, int minorColorIndex) {
+    return majorColorIndex * 5 + minorColorIndex + 1;
+}
+
 int printColorMap() {
     const char* majorColor[] = {"White", "Red", "Black", "Yellow", "Violet"};
     const char* minorColor[] = {"Blue", "Orange", "Green", "Brown", "Slate"};
     int i = 0, j = 0;
     for(i = 0; i < 5; i++) {
         for(j = 0; j < 5; j++) {
-            std::cout << i * 5 + j << " | " << majorColor[i] << " | " << minorColor[i] << "\n";
+            std::cout << getColorPairNumber(i, j) << " | " << majorColor[i] << " | " << minorColor[i] << "\n";
         }
     }
     return i * j;
 }
+}
 
-int main() {
-    int result = printColorMap();
+namespace ColorMapTests {
+void TestPrintColorMap()

```

Comment:
Name of the function is TestPrintColorMap, but it's only proving the computation of the pair number?